auto_nag/utils.py

```diff
@@ -609,3 +619,14 @@ def bug_handler(bug, data):
 
 def nice_round(val):
     return int(round(100 * val))
+
+
+def sort_bugs_by_importance(action, bug):
```

Comment:
Nit: the function isn't really sorting but returning a key, so I'd rename it.

Also, I'd somehow note in the name that it's sorting bugs and actions, not bugs (the needinfo is not a property of a bug)

Reply:
Fixed in c92e238ab5e677fa807a74438a540369167f032d